### PR TITLE
RNMT-1278 (iOS) Code convergence - Increment log timer 

### PR DESCRIPTION
**Pull request by vmfo22**

### CODE REVIEW

File: Pod/Classes/PURBufferedOutput.m [hunk 1]
```diff
@@ -50,11 +50,16 @@ - (void)dealloc
 - (void)setUpTimer
 {
     [self.timer invalidate];
-    self.timer = [NSTimer timerWithTimeInterval:1.0
-                                         target:self
-                                       selector:@selector(tick)
-                                       userInfo:nil
-                                        repeats:YES];
+    if(self.flushInterval <= 0){
```
Comment (EiyuuZack) [inline Pod/Classes/PURBufferedOutput.m:53]: Why this protection for the flush interval? Are we expecting it to be non-positive at some point?

File: Pod/Classes/PURBufferedOutput.m [hunk 2]
```diff
@@ -116,15 +121,13 @@ - (void)resume
 - (void)suspend
 {
     [self.timer invalidate];
-
+
```
Comment (EiyuuZack) [inline Pod/Classes/PURBufferedOutput.m:124]: #killallthetrailingwhitespace

**Review by EiyuuZack — CHANGES_REQUESTED** [review]
Nitpicking

**Review by bgracio — APPROVED** [review]

File: Pod/Classes/PURBufferedOutput.m (hunk 1 above)
Comment (vmfo22) [reply Pod/Classes/PURBufferedOutput.m:53]: This setting is optional, in case that is not configured, needs to be protected to set a valid value.

**Review by luissilvaos — APPROVED** [review]

File: Pod/Classes/PURBufferedOutput.m (hunk 1 above)
Comment (EiyuuZack) [reply Pod/Classes/PURBufferedOutput.m:53]: Fair enough.

**Review by EiyuuZack — APPROVED** [review]
🚢

**Review by luissilvaos — APPROVED** [review]

**Pull request merged**